workload/cfg_icni2_serving_resource_init.yml

```diff
@@ -72,6 +72,25 @@ jobs:
       - objectTemplate: objectTemplates/cm_frr.yaml
         replicas: 1
 
+  - name: create-secrets-job
+    jobType: create
+    jobIterations: {{ multiply 35 .SCALE }}
+    qps: 1
+    burst: 1
+    namespacedIterations: true
+    cleanup: false
+    namespace: serving-ns
+    podWait: false
+    waitWhenFinished: false
+    waitFor: ["Secret"]
+    verifyObjects: true
+    errorOnVerify: false
+    jobIterationDelay: 1s
+    jobPause: 2s
+    objects:
+      - objectTemplate: objectTemplates/secret_kubeconfig.yaml
```

Comment:
Where is this file?

Reply:
It is generated dynamically by https://github.com/redhat-performance/web-burner/blob/cace36d2fd6bf68b0022250d826df227fcd8caeb/create_icni2_workload.sh#L29

Reply:
Ah, nice.